index.html

```diff
@@ -7,42 +7,108 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
-      text-align: center;
+      text-align: center;                 
+      font-size: 24px;
+      font-weight: bold;
+      color: #2f4f4f;
+            }
     }
   </style>
+    <style>
```

Comment:
All of these rules can be inside a single STYLE element (and should be). Also note that inside each STYLE element, we have an extra closing curly-brace (`}`). The browser's helpful enough to ignore that for us, but it could cause bugs if we tried to add additional rules (Javascript would throw a fit).